grizli/prep.py

```diff
@@ -153,7 +187,7 @@ def fresh_flt_file(file, preserve_dq=False, path='../RAW/', verbose=True, extra_
         extra_msg = ' / flat: {0}'.format(flat_file)
 
         flat_im = pyfits.open(os.path.join(os.getenv('iref'), flat_file))
-        flat = flat_im['SCI'].data[5:-5, 5:-5]
+        flat = flat_im['SCI'].data [5:-5, 5:-5]
```

Comment:
remove space. And these 5s are only appropriate for WFC3/IR.  Does JWST processing get to this point?

Reply:
Fixed: 94713d82b411007ad54fb5022e913b17e1fd3287